app/controllers/cinema_halls_controller.rb

```diff
@@ -1,4 +1,48 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+  def index
```

Comment:
You haven't closed another PR and this changes shouldn't be included in this PR :) Here are some changes that are not connected with rspec branch